cmd/mothership/mothership/start/http.go

```diff
@@ -917,6 +892,19 @@ func updateOperationStateAndRetryID(o *Options, schedulingID, correlationID, ret
 	return err
 }
 
+func updateOperationStateAndRetryIDAndProcessingDuration(o *Options, schedulingID, correlationID, retryID string, state model.OperationState, processingDuration int, reason ...string) error {
+	err := updateOperationStateAndProcessingDuration(o, schedulingID, correlationID, state, processingDuration, reason...)
+	if err != nil {
+		return err
+	}
+	err = o.Registry.ReconciliationRepository().UpdateOperationRetryID(schedulingID, correlationID, retryID)
```

Comment:
Do we need this function? Can't we just add this line also to `updateOperationStateAndProcessingDuration` ?

Reply:
I left it here, since they are dealing with different tables, and ot keep the function names as "short" as possible 😅 
But we could also add it there yes